docs/src/engines/eon.rst

```diff
@@ -0,0 +1,66 @@
+.. _engine-eon:
+
+eOn
+===
+
+.. list-table::
+   :header-rows: 1
+
+   * - Official website
+     - How is metatomic supported?
+   * - https://theochemui.github.io/eOn/
+     - In the official Github version
+
+
+Supported model outputs
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Only the :ref:`energy <energy-output>` output is supported in eOn, as a custom
+``Potential``. This allows running methods, including:
+
+- Saddle search methods
+  - Single ended (dimer method, GPR accelerated dimer)
+  - Double ended (Nudged Elastic Band with energy weighted strings)
+- Adaptive Kinetic Monte Carlo (aKMC) for long time scale simulations
+
+With the engine integration, it is possible to run these with interatomic
+potentials in ``metatomic`` format; distributing the calculation on GPUs as
+well.
+
+How to install the code
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The metatomic interface is part of eOn was completed in `PR-206`_. Please refer
+to latest `eOn documentation`_ about how to install it.
+
+.. _eOn documentation: https://theochemui.github.io/eOn/install/metatomic.html
+.. _PR-206: https://github.com/TheochemUI/eOn/pull/206
```

Comment:
```suggestion
Please refer to latest `eOn documentation`_ about how to install it.

.. _eOn documentation: https://theochemui.github.io/eOn/install/metatomic.html
```

I don't think users will care about the PR that added the code =). Or do you want to say "this is not released yet, please see the instructions to build from source"?